README.md

```diff
@@ -25,9 +23,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Run CJ Action
-        uses: foreai-co/cj-action@v1.0.3
+        uses: foreai-co/cj-action@v1.0.4
+        id: run_cj
         with:
-          instructions: 'Go to my website and try logging in.'
-          url: 'https://staging.my_website.com'
-          name: 'Login test'
+          test_id: 'my-test-id'
```

Comment:
the example test id `my-test-id` is a bit confusing (was to me), when test ids really look like `66b4b44f2b7aac23867efe2c` - once we have a test suite, it would be nice to be able to target testsuites by their names, rather than ids.